packages/webapp/src/hooks/useLogs.tsx

```diff
@@ -88,5 +101,55 @@ export function useSearchMessages(env: string, body: SearchMessages['Body']) {
         }
     }, [env, body.operationId, body.limit, body.states, body.search]);
 
-    return { data, error, loading };
+    function trigger() {
+        if (!loading) {
+            void fetchData();
+        }
+    }
+
+    return { data, error, loading, trigger };
+}
+
+export function useSearchFilters(enabled: boolean, env: string, body: SearchFilters['Body']) {
+    const [loading, setLoading] = useState<boolean>(false);
+    const [data, setData] = useState<SearchFilters['Success']>();
+    const [error, setError] = useState<SearchFilters['Errors']>();
+
+    async function fetchData() {
+        setLoading(true);
+        try {
+            const res = await fetch(`/api/v1/logs/filters?env=${env}`, {
```

Comment:
useSWR not supporting POST with cache is really annoying. Even regular POST calls are barely supported. 
I'll need to look for a solution because it's spamming the backend for no reason, especially when playing with filters

Reply:
Ugh